core/src/main/java/io/opencensus/stats/MeasurementMap.java

```diff
@@ -18,34 +18,34 @@
 import java.util.NoSuchElementException;
 
 /**
- * A map from {@link MeasurementDescriptor}'s to measured values.
+ * A map from {@link Measure}'s to measured values.
  */
 public final class MeasurementMap implements Iterable<MeasurementValue> {
```

Comment:
Similarly MeasurementValue should probably be MeasureValue. As discussed, StatsEvent or Measurement may also be reasonable alternatives.